jest-setup.js

```diff
@@ -0,0 +1,3 @@
+import fetch from "jest-fetch-mock"; // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
Is there a reason this is an extraneous dependency? Surely it could be declared in devDependencies? I feel like I must be missing context...

Reply:
It can't just be put into devDependencies as we actually need to overwrite the fetch module at runtime, which this is doing...

...BUT we only need to do that where we're testing the `await fetch('data.json')` stuff - which is Simorgh rather than Psammead specific 🤔 so again, can remove! Good spot.